main.go

```diff
@@ -228,8 +228,10 @@ func main() {
 		},
 	}
 
-	mgrConfig.Cache.DefaultNamespaces = map[string]ctrlcache.Config{
-		watchNamespace: {},
+	if watchNamespace != "" {
+		mgrConfig.Cache.DefaultNamespaces = map[string]ctrlcache.Config{
+			watchNamespace: {},
```

Comment:
```suggestion
			watchNamespace: ctrlcache.Config{},
```

For consistency with all the other controllers.